djangocms_versioning/admin/using_versioning.py

```diff
@@ -0,0 +1,222 @@
+from django.conf.urls import url
+from django.contrib.admin.utils import flatten_fieldsets, unquote
+from django.db import transaction
+from django.shortcuts import redirect, render
+from django.urls import reverse
+from django.utils.translation import ugettext_lazy as _
+
+from cms import api
+from cms.extensions import extension_pool
+from cms.models import PageContent
+
+from .. import versionables
+from ..compat import DJANGO_GTE_21
+from ..forms import DuplicateForm
```

Comment:
I don't like relative imports that scale over a a dir. Personal preference so not a big deal.

Reply:
Fixed